src/modules/chat/index.js

```diff
@@ -13,6 +13,7 @@ const splitChat = require('../split_chat');
 
 const EMOTE_STRIP_SYMBOLS_REGEX = /(^[~!@#$%\^&\*\(\)]+|[~!@#$%\^&\*\(\)]+$)/g;
 const MENTION_REGEX = /^@([a-zA-Z\d_]+)$/;
+const STEAM_LOBBY_JOIN_REGEX = /^steam:\/\/joinlobby\/[0-9]*\/[0-9]*\/[0-9]*$/;
```

Comment:
You can replace [0-9] with \d, and also use + instead of * since these IDs are guaranteeds non-empty

Reply:
replaced [0-9]* with \d+